lib/v2/screens/profile_screens/citizenship/citizenship_screen.dart

```diff
@@ -18,48 +23,25 @@ class CitizenshipScreen extends StatefulWidget {
 }
 
 class _CitizenshipScreenState extends State<CitizenshipScreen> with TickerProviderStateMixin {
+  static const int required_planted_seeds = 200;
+  static const int required_reputation = 50;
+  static const int required_seeds_transactions = 5;
+  static const int required_visitors_invited = 3;
+  static const int required_resident_invited = 1;
+  static const int required_account_age = 59;
```

Comment:
These are constants of the requirements to be a citizen,I do not know if it is okay to defined them here. Any suggestion?

Reply:
they seem to be more of a model than a state?

Reply:
Amm can you be more clear here, I didnt' get it, should I turn those constants into a model?

Reply:
its fine here for now. As long as they are only used here. 
What i want to avoid is needing these values somewhere else, and recreating them there. 

If they are unique to this page, fine here, If they are shared, then find a shared place to put them.